components/Profile/Settings.js

```diff
@@ -30,41 +30,59 @@ const Settings = ({ user, isEditing, onChange, values, errors, dirty, t }) => {
         <br /><br />
       </Label>
     }
-    <div style={{ opacity: user.isAdminUnlisted ? 0.5 : 1 }}>
-      <Checkbox
-        checked={values.isListed}
-        style={{ display: 'block' }}
-        disabled={(
-          !(
-            user.isListed ||
-            (
-              (user.statement || values.statement) &&
-              (user.portrait || values.portrait)
-            )
-          ) ||
-          (!user.isListed && !user.isEligibleForProfile)
-        )}
-        onChange={(_, checked) => {
-          onChange({
-            values: {
-              isListed: checked
-            }
-          })
-        }}
-      >
-        {t('profile/settings/isListed/label')}
-      </Checkbox>
-      <br />
-      <Label>{t.elements(`profile/settings/isListed/${values.isListed}/note`, {
-        communityLink: <Link
-          key='communityLink'
-          route='community'
-          passHref>
-          <A target='_blank'>{t('profile/settings/privacy/communityLink')}</A>
-        </Link>
-      })}</Label>
+    <ListedCheckbox user={user} values={values} onChange={onChange} />
+    <br />
+    <PublicCheckbox user={user} values={values} onChange={onChange} />
+
+    <div style={{ marginTop: 5 }}>
+      <UsernameField
+        user={user}
+        values={values}
+        errors={errors}
+        onChange={onChange} />
     </div>
+  </div>
+}
+
+export const ListedCheckbox = withT(({ user, values, onChange, t }) => (
+  <div style={{ opacity: user.isAdminUnlisted ? 0.5 : 1 }}>
+    <Checkbox
+      checked={values.isListed}
+      style={{ display: 'block' }}
+      disabled={(
+        !(
+          user.isListed ||
+          (
+            (user.statement || values.statement) &&
+            (user.portrait || values.portrait)
+          )
+        ) ||
+        (!user.isListed && !user.isEligibleForProfile)
+      )}
+      onChange={(_, checked) => {
+        onChange({
+          values: {
+            isListed: checked
+          }
+        })
+      }}
+    >
+      {t('profile/settings/isListed/label')}
+    </Checkbox>
     <br />
```

Comment:
```suggestion
     <br style={{ clear: 'left' }} />
```
_preexisting condition but let's fix it_